code/datums/abilities/revenant.dm

```diff
@@ -456,6 +462,7 @@
 	special_screen_loc = "NORTH-1,WEST+4"
 
 	cast(atom/target)
+		playsound(target.loc, "sound/voice/wraith/revfocus.ogg", 120, 0)
```

Comment:
is 120 even a valid number

Reply:
Believe it or not, yes. Something weird happens with this noise, maybe its compression but it sounds much quieter in game than in any playback  and this gets it around the level I'd like.